runtime-testsuite/test/org/antlr/v4/test/runtime/go/BaseGoTest.java

```diff
@@ -165,6 +165,9 @@ private static void cacheGoRuntime(File tmpPackageDir) throws Exception {
 	}
 
 	private static void copyFile(File source, File dest) throws IOException {
+		if ( source.isDirectory() ) {
```

Comment:
Hey @millergarym, The best thing to do is to run `gofmt` on the file. BTW, in Go, you don't need parens around the conditional expression. `if expr { ... }` should work.